WSVC Warning C4456: declaration of 'side_pk_q1' hides previous local declaration

Compiling with boost 1.75.0, MSVC 2019 Version 16.9.2 with compiler options `<WarningLevel>Level4</WarningLevel>` and `<LanguageStandard>stdcpplatest</LanguageStandard>`, I get the following warning:

```
22>\boost_1_75_0\boost\geometry\algorithms\detail\overlay\get_turn_info.hpp(731,1): warning C4456: declaration of 'side_pk_q1' hides previous local declaration
22>\boost_1_75_0\boost\geometry\algorithms\detail\overlay\get_turn_info.hpp(608,19): message : see declaration of 'side_pk_q1'
22>\boost_1_75_0\boost\geometry\algorithms\detail\overlay\get_turn_info.hpp(1414): message : see reference to function template instantiation 'void boost::geometry::detail::overlay::touch<TurnInfo>::apply<UniqueSubRange1,UniqueSubRange2,boost::geometry::segment_intersection_points<TurnPoint,boost::geometry::segment_ratio<boost::long_long_type>>,boost::geometry::policies::relate::direction_type,boost::geometry::detail::overlay::side_calculator<boost::geometry::cartesian_tag,boost::geometry::detail::overlay::robust_subrange_adapter<boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,UniqueSubRange1,RobustPolicy>,boost::geometry::detail::overlay::robust_subrange_adapter<boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,UniqueSubRange1,RobustPolicy>,boost::geometry::strategy::side::side_by_triangle<CalculationType>>,UmbrellaStrategy>(const UniqueSubRange1 &,const UniqueSubRange2 &,TurnInfo &,const IntersectionInfo &,const DirInfo &,const SideCalculator &,const UmbrellaStrategy &)' being compiled
22>        with
22>        [
22>            TurnInfo=turn_info,
22>            UniqueSubRange1=boost::geometry::detail::get_turns::unique_sub_range_from_section<true,boost::geometry::section<box_type,2>,tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::ever_circling_iterator<boost::geometry::closing_iterator<const ring_type>>,strategy_type,boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>>,
22>            UniqueSubRange2=boost::geometry::detail::get_turns::unique_sub_range_from_section<true,boost::geometry::section<box_type,2>,tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::ever_circling_iterator<boost::geometry::closing_iterator<const ring_type>>,strategy_type,boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>>,
22>            TurnPoint=tc::geo::no_adl::pointWORKAROUND<double>,
22>            RobustPolicy=boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>,
22>            CalculationType=void,
22>            UmbrellaStrategy=strategy_type,
22>            IntersectionInfo=boost::geometry::segment_intersection_points<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::segment_ratio<boost::long_long_type>>,
22>            DirInfo=boost::geometry::policies::relate::direction_type,
22>            SideCalculator=boost::geometry::detail::overlay::side_calculator<boost::geometry::cartesian_tag,boost::geometry::detail::overlay::robust_subrange_adapter<boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,boost::geometry::detail::get_turns::unique_sub_range_from_section<true,boost::geometry::section<box_type,2>,tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::ever_circling_iterator<boost::geometry::closing_iterator<const ring_type>>,strategy_type,boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>>,boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>>,boost::geometry::detail::overlay::robust_subrange_adapter<boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,boost::geometry::detail::get_turns::unique_sub_range_from_section<true,boost::geometry::section<box_type,2>,tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::ever_circling_iterator<boost::geometry::closing_iterator<const ring_type>>,strategy_type,boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>>,boost::geometry::detail::robust_policy<tc::geo::no_adl::pointWORKAROUND<double>,boost::geometry::model::point<__int64,2,boost::geometry::cs::cartesian>,double>>,boost::geometry::strategy::side::side_by_triangle<void>>
22>        ]
```

--- a/include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp
+++ b/include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp
@@ -693,8 +693,6 @@ struct touch : public base_turn_handler
             // If p turns into direction of qi (1,2)
             if (side_pk_p == side_qi_p1)
             {
-                int const side_pk_q1 = has_pk ? side.pk_wrt_q1() : 0;
-
                 // Collinear opposite case -> block P
                 if (side_pk_q1 == 0)
                 {

--- a/include/boost/geometry/strategies/distance/geographic.hpp
+++ b/include/boost/geometry/strategies/distance/geographic.hpp
@@ -238,12 +238,12 @@ struct strategy_converter<strategy::distance::detail::geographic_cross_track<FP,
         auto distance(Geometry1 const&, Geometry2 const&,
                       std::enable_if_t
                       <
-                            util::is_pointlike<Geometry1>::value
-                                && util::is_segmental<Geometry2>::value
-                         || util::is_segmental<Geometry1>::value
-                                && util::is_pointlike<Geometry2>::value
-                         || util::is_segmental<Geometry1>::value
-                                && util::is_segmental<Geometry2>::value
+                            (util::is_pointlike<Geometry1>::value
+                                && util::is_segmental<Geometry2>::value)
+                         || (util::is_segmental<Geometry1>::value
+                                && util::is_pointlike<Geometry2>::value)
+                         || (util::is_segmental<Geometry1>::value
+                                && util::is_segmental<Geometry2>::value)
                       > * = nullptr) const
         {
             return strategy::distance::detail::geographic_cross_track

--- a/include/boost/geometry/util/series_expansion.hpp
+++ b/include/boost/geometry/util/series_expansion.hpp
@@ -64,7 +64,8 @@ namespace boost { namespace geometry { namespace series_expansion {
     {
         CT eps2 = math::sqr(eps);
         CT t;
-        switch (SeriesOrder/2) {
+        switch (SeriesOrder/2)
+        {
         case 0:
             t = CT(0);
             break;
@@ -109,7 +110,8 @@ namespace boost { namespace geometry { namespace series_expansion {
     {
         CT const eps2 = math::sqr(eps);
         CT t;
-        switch (SeriesOrder/2) {
+        switch (SeriesOrder/2)
+        {
         case 0:
             t = CT(0);
             break;
@@ -152,7 +154,8 @@ namespace boost { namespace geometry { namespace series_expansion {
     template <typename Coeffs, typename CT>
     inline void evaluate_coeffs_A3(Coeffs &c, CT const& n)
     {
-        switch (int(Coeffs::static_size)) {
+        switch (int(Coeffs::static_size))
+        {
         case 0:
             break;
         case 1:
@@ -223,7 +226,8 @@ namespace boost { namespace geometry { namespace series_expansion {
     {
         CT eps2 = math::sqr(eps);
         CT d = eps;
-        switch (int(Coeffs::static_size) - 1) {
+        switch (int(Coeffs::static_size) - 1)
+        {
         case 0:
             break;
         case 1:
@@ -322,7 +326,8 @@ namespace boost { namespace geometry { namespace series_expansion {
     {
         CT const eps2 = math::sqr(eps);
         CT d = eps;
-        switch (int(Coeffs::static_size) - 1) {
+        switch (int(Coeffs::static_size) - 1)
+        {
         case 0:
             break;
         case 1:
@@ -421,7 +426,8 @@ namespace boost { namespace geometry { namespace series_expansion {
     {
         CT const eps2 = math::sqr(eps);
         CT d = eps;
-        switch (int(Coeffs::static_size) - 1) {
+        switch (int(Coeffs::static_size) - 1)
+        {
         case 0:
             break;
         case 1:
@@ -517,12 +523,11 @@ namespace boost { namespace geometry { namespace series_expansion {
     */
     template <size_t SeriesOrder, typename Coeffs, typename CT>
     inline void evaluate_coeffs_C3x(Coeffs &c, CT const& n) {
-        size_t const coeff_size = Coeffs::static_size;
-        size_t const expected_size = (SeriesOrder * (SeriesOrder - 1)) / 2;
-        BOOST_GEOMETRY_ASSERT((coeff_size == expected_size));
+        BOOST_GEOMETRY_ASSERT((Coeffs::static_size == (SeriesOrder * (SeriesOrder - 1)) / 2));
 
-        const CT n2 = math::sqr(n);
-        switch (SeriesOrder) {
+        CT const n2 = math::sqr(n);
+        switch (SeriesOrder)
+        {
         case 0:
             break;
         case 1:

